UI.md

```diff
@@ -2,7 +2,7 @@
 
 1. Avoid put more than 5 React components in a single file. Use separate files for big components or components that are used in multiple places.
 2. Avoid files with only a few lines of code that are overly simple, rather try group them meaningfully.
-3. Put multiline styles in the top of the file.
+3. Put multiline styles in the top of the file. [not sure i agree with this, inline is easier unless you're repeating the style]
```

Comment:
Hmm, yeah this was for styled components more than tailwind, can delete this line.